code/game/machinery/computer/law.dm

```diff
@@ -301,7 +301,10 @@
 /obj/machinery/computer/aiupload/update_icon()
 	..()
 	overlays = 0
-
+	
+	if(stat & (BROKEN | NOPOWER))
```

Comment:
but what happens if the AI dies while unpowered?

Reply:
the overlay wont appear on the upload if the upload is broken/unpowered. If the AI is dead/out of power then the overlay will reflect that

Reply:
nevermind i was too sleep-driven when i wrote that.